### PR TITLE
Update to use latest fluentd

### PR DESCRIPTION
**Pull request by jcantrill**
This PR updates master:

* fluent v1.11

### CODE REVIEW

**Comment by jcantrill** [comment]
cc @ewolinetz @alanconway

**Comment by jcantrill** [comment]
fixes https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/issues/240

File: Gemfile.lock [hunk 1]
```diff
@@ -1,8 +1,8 @@
 PATH
   remote: .
   specs:
-    fluent-plugin-kubernetes_metadata_filter (2.4.6)
-      fluentd (>= 0.14.0, < 1.10)
+    fluent-plugin-kubernetes_metadata_filter (2.5.0)
+      fluentd (>= 1.10)
```
Comment (ewolinetz) [inline Gemfile.lock:5]: why not just adjust the range to include `1.11` ?

Comment (jcantrill) [reply Gemfile.lock]: restrict upper bound to less then 1.11

**Pull request merged**